developer-docs-site/docs/guides/local-development-network.md

```diff
@@ -230,7 +230,26 @@ Make sure the socket for connecting to Docker is present on your machine in the
 /var/run/docker.sock
 ```
 
-If you're on Mac or Windows, we recommend you use Docker Desktop rather than installing Docker via a package manager (e.g. Homebrew or Choco).
+If it doesn't, you can find where it is like this:
+```
+$ docker context inspect | grep Host
+  "Host": "unix:///Users/dport/.docker/run/docker.sock",
+```
+
+Then make a symlink to it in the expected location:
+```
+sudo ln -s /Users/dport/.docker/run/docker.sock /var/run/docker.sock
+```
+
+Alternatively, run the CLI like this to tell it where the socket is:
+```
+DEFAULT_SOCKET=/Users/dport/.docker/run/docker.sock aptos node run-local-testnet --with-indexer-api
+```
+
+Note: If you're on Mac or Windows, we recommend you use Docker Desktop rather than installing Docker via a package manager (e.g. Homebrew or Choco).
```

Comment:
Please move this to the top of Prerequisites.  Where you note them to install Docker Desktop first rather than get all the way to the bottom and realize this.